.gitignore

```diff
@@ -10,4 +10,6 @@ coverage/
 coverage.txt
 *.swp
 *.swo
-testworld/peerconfigs/*
\ No newline at end of file
+testworld/peerconfigs/*
+testworld/configs/kovan.json
```

Comment:
these are just files I have for kovan in local that I don't want to accidentally commit to git

Reply:
they are not in the codebase, because of the ignore line :) . I had two options either configure some place to pick kovan.json from `out of repo` location or do this ignore, i feel this is sensible.

Reply:
they are codebase, as the .gitignore is part of the code repo. Is the same as me storing a picture of my cat in the repo locally and wanting to ignore it for everyone else that uses it. I think is more sensible to pull from outside the repo if it just for testing something particular to your use needs only.

Reply:
I removed, since you are very picky on this :) , but the .gitignore already contains very specific files like idea or vscode configs in many projects, thats a very standard use thats not common to everyone.